pinc/theme.inc

```diff
@@ -179,12 +179,18 @@ function html_logobar()
             "class" => "fas fa-question fa-2x",
             "url" => get_faq_url("faq_central.php"),
         ];
+        $icon_links[] = [
+            "name" => _("Search"),
+            "class" => "fas fa-search fa-2x",
```

Comment:
The current HTML output does not change the mouse on hover, so I wasn't aware this search icon was interact-able at first. Could we make the cursor a hand on hover link an anchor?

Reply:
Good catch - I'll fix this.